src/run.js

```diff
@@ -73,10 +74,25 @@ const loadConfig = testPath =>
 
 // Run tests on a given file
 const run = ({ testPath, config, globalConfig }) => {
-  const schema = require(testPath)
+  const { transform } = config
+
+  let schema
+  let fileContent = fs.readFileSync(testPath, { encoding: 'utf-8' })
+
+  if (
+    !config.transformIgnorePatterns
+      .map(x => new RegExp(x))
+      .find(x => x.test(testPath))
+  ) {
+    const transformer = transform.find(x => new RegExp(x[0]).test(testPath))
+
+    if (transformer) {
+      const transformerObj = require(transformer[1])
```

Comment:
It looks like transforms can be provided [as an array as well](https://jestjs.io/docs/en/configuration#transform-objectstring-pathtotransformer--pathtotransformer-object), if we're going to support the jest config we should deal with that too.